src/linear_algebra/dfinsupp.lean

```diff
@@ -375,6 +405,12 @@ lemma independent_iff_dfinsupp_lsum_injective (p : ι → submodule R N) :
   independent p ↔ function.injective (lsum ℕ (λ i, (p i).subtype)) :=
 ⟨independent.dfinsupp_lsum_injective, independent_of_dfinsupp_lsum_injective p⟩
 
+/-- A family of additive subgroups over an additive group are independent if and only iff
```

Comment:
```suggestion
/-- A family of additive subgroups over an additive group are independent if and only if
```
(I probably wouldn't bother CI with this unless you're making some other changes)